packages/client-python/src/rocketride/core/transport_websocket.py

```
@@ -242,7 +243,8 @@ async def _receive_loop(self) -> None:
 
                 # Process each message in its own task so others can be processed concurrently
                 task = asyncio.create_task(self._receive_data(data))
-                self._message_tasks.add(task)
+                async with self._message_tasks_lock:
+                    self._message_tasks.add(task)
                 task.add_done_callback(lambda t: self._message_tasks.discard(t))
```

Ruff (0.15.7)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)